src/lib/pickles_base/domain.mli

```diff
@@ -1,4 +1,4 @@
-(* Domain specification *)
+(** Wrapping integers for computing FFT domain *)
```

Comment:
Part of number theoretic transform?

Reply:
You're right, better to use NTT instead of FFT.